dts/arm/nordic/nrf54h20_cpuapp.dtsi

```diff
@@ -59,3 +59,23 @@ wdt011: &cpuapp_wdt011 {};
 &grtc {
 	interrupts = <109 NRF_DEFAULT_IRQ_PRIORITY>;
 };
+
+/ {
+	power-states {
```

Comment:
power states must be within cpu node, check other cases

Reply:
Changed from root node to cpus like here
https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/arm/silabs/efr32bg2x.dtsi#L52
Is this what You meant?